sound/soc/sof/ipc4.c

```diff
@@ -653,9 +653,24 @@ static int sof_ipc4_ctx_save(struct snd_sof_dev *sdev)
 	return sof_ipc4_set_core_state(sdev, SOF_DSP_PRIMARY_CORE, false);
 }
 
+static int sof_ipc4_set_pm_state(struct snd_sof_dev *sdev, u32 flags)
```

Comment:
why mention reuse of flags. They may have similar meaning across IP3 and IPC4 but if you mention reuse it will raise questions.

Reply:
sure, it is useless to mention it.

Reply:
I guess it is up to the caller to pass down the correct flags if they are different across the supported IPC versions?